### PR TITLE
Update sbt to 1.5.0-RC2

### PR DESCRIPTION
**Pull request by sh0hei**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -72,9 +72,10 @@ val plugin = project.settings(
   crossVersion := CrossVersion.full,
   libraryDependencies ++= Seq(
     scalaOrganization.value % (
-      if (isDotty.value)
+      if (scalaVersion.value.startsWith("3.0.0"))
```
Comment (kubukoz) [inline build.sbt:75]: hmm isn't `isDotty` supported in 1.5.0-RC1 and above?

Comment (sh0hei) [reply build.sbt:75]: `isDotty` is a feature provided by sbt-dotty, and I think sbt-dotty is no longer needed after sbt-1.5.0-RC2.

- https://github.com/sbt/sbt/releases/tag/v1.5.0-RC2
  > Scala 3 support
sbt 1.5.0 adds built-in Scala 3 support, contributed by Scala Center. Main implementation was done by Adrien Piquerez (@adpi2) based on EPFL/LAMP's sbt-dotty.

- https://github.com/lampepfl/dotty/blob/master/sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala#L184-L187
  ```scala
  val deprecatedVersion = ">=1.5.0-RC2"
  val logger = sLog.value
  if (VersionNumber(sbtV).matchesSemVer(SemanticSelector(deprecatedVersion)))
    logger.warn(s"The sbt-dotty plugin is no longer neeeded with sbt >= 1.5, please remove it from your build.")
  ```

Comment (kubukoz) [reply build.sbt:75]: strange that they didn't leave the task in :/ can you double-check that on 1.5.0 final? (it's now on master)

**Comment by kubukoz** [comment]
covered by #23 an #17

**Comment by adpi2** [comment]
@kubukoz `sbt-dotty` is not needed anymore in this build.

**Comment by kubukoz** [comment]
right. I'll accept a PR that removes it separately straight away ;)

**Pull request closed without merging**